### PR TITLE
fix: skeleton cannot display children

### LINKED ISSUE #4936: 🐛[BUG] ant design 升级 4.19.4 版本，ProList 列表无法正常展示！

### LINKED ISSUE #4939: 🐛[BUG] pro-list升级后metas中的部分内容不显示
升级前：

升级后：

### PR DESCRIPTION
**Pull request by Dunqing**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

related pr: #34699

fix: https://github.com/ant-design/pro-components/issues/4939
fix: https://github.com/ant-design/pro-components/issues/4936
### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix skeleton to unable to render childten of non-react element |
| 🇨🇳 Chinese | 修复skeleton不渲染非react element类型的children问题 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by chenshuai2144** [comment]
这个问题好奇怪，会导致 prolist 不能用吗？

**Comment by 1247748612** [comment]
> 这个问题好奇怪，会导致 prolist 不能用吗？

对

因为现在的实现多了个isValidElement的判断，在ProList中使用时传递的chilren只是个ReactNode

这算是breaking change以前的实现是正常渲染的

**Comment by chenshuai2144** [comment]
得看看为什么。是不是为了解某些bug？

**Comment by 1247748612** [comment]
> 看上去确实不知道为啥要加这个判断：https://github.com/ant-design/ant-design/commit/9af935f7d3cce846be45f8e4c29ed7bf45672048#diff-06af9cb647c92b123b921f7be2e4d7738eb2914d066a832bb4a5ac0b580a3c03R164
> 
> 
> 
> @heiyu4585

我猜是类型问题，返回ReactNode其他地方一定要个ReactElement，我尝试加个React.Fragement然后lint弹个警告提示不需要包裹直接使用children就行。最后我强制类型为ReactElement了

**Comment by heiyu4585** [comment]
> 我猜是类型问题，返回ReactNode其他地方一定要个ReactElement，我尝试加个React.Fragement然后lint弹个警告提示不需要包裹直接使用children就行。最后我强制类型为ReactElement了

当时是解决类型问题,React.Fragemen只有单个元素,也过不去lint

File: components/skeleton/Skeleton.tsx [hunk 1]
```diff
@@ -161,7 +161,7 @@ const Skeleton = (props: SkeletonProps) => {
       </div>
     );
   }
-  return React.isValidElement(children) ? children : null;
+  return children as React.ReactElement;
```
Comment (afc163) [inline components/skeleton/Skeleton.tsx:164]:
```suggestion
-  return children as React.ReactElement;
+  return <>{children}</>;
```

这样简单点。

**Pull request merged**